test/src/app.js

```diff
@@ -1,4 +1,183 @@
+import React from 'react';
+// import IField, { CARD_TYPE, CVV_TYPE } from '@cardknox/react-ifields';
+import CardknoxApplePay from './apple-pay';
+import * as lib from './lib';
 
+export default class App extends React.Component {
+    constructor(props) {
+        super(props);
+        const account = {
+            xKey: "",
+            xSoftwareName: "react-cardknox-ifields",
+            xSoftwareVersion: "1.0.0"
+        };
+        const threeds = {
+            enable3DS: false,
+            waitForResponse: false,
+            waitForResponseTimeout: undefined,
+            amount: 0,
+            month: "01",
+            year: "2020"
+        };
+        const ccoptions = {
+            placeholder: 'Card Number',
+            enableLogging: true,
+            autoFormat: true,
+            autoFormatSeparator: ' ',
+            autoSubmit: false,
+            iFieldstyle: {
+
+            },
+            iFrameStyle: {
+
+            }
+        };
+        const cvvoptions = {
+            placeholder: 'CVV',
+            enableLogging: true,
+            autoFormat: true,
+            autoFormatSeparator: ' ',
+            autoSubmit: false,
+            iFieldstyle: {
+
+            },
+            iFrameStyle: {
+
+            }
+        };
+        this.state = {
+            account,
+            threeds,
+            ccoptions,
+            cvvoptions,
+            issuer: ''
+        };
+        this.cvvRef = React.createRef();
+        this.cardRef = React.createRef();
+    }
+
+    get getApplePayProperties()  {
+        return {
+            merchantIdentifier: 'merchant.aptest.cardknoxdev.com',
+            requiredShippingContactFields: ['postalAddress']
+        }
+    }
+
+    render() {
+        return (
+            <>
+
+                {/* <IField
```

Comment:
Should this be removed?
Also, where is the sample for regular ifields?

Reply:
Discussed